ownCloudAppShared/AppLock/PasscodeSetupCoordinator.swift

```diff
@@ -127,6 +142,25 @@ public class PasscodeSetupCoordinator {
 		}
 	}
 
+	public func showNumberOfDigitsSetup() {
```

Comment:
@hosy method name.. may be `showDigitsCountSelectionUI() `?

Reply:
I changed the naming. Thanks!